lib/compiler/defaults/swc-defaults.ts

```diff
@@ -25,14 +29,14 @@ export const swcDefaultsFactory = (tsOptions: ts.CompilerOptions) => {
       swcrc: true,
     },
     cliOptions: {
-      outDir: 'dist',
-      filenames: ['src'],
-      sync: false,
-      extensions: ['.js', '.ts'],
+      outDir: swcCliOptions?.outDir || tsOptions.outDir || 'dist',
+      filenames: swcCliOptions?.filenames || ['src'],
+      sync: swcCliOptions?.sync || false,
+      extensions: swcCliOptions?.extensions || ['.js', '.ts'],
       watch: false,
-      copyFiles: false,
-      includeDotfiles: false,
-      quiet: false,
+      copyFiles: swcCliOptions?.copyFiles || false,
+      includeDotfiles: swcCliOptions?.includeDotfiles || false,
+      quiet: swcCliOptions?.quiet || false,
```

Comment:
Wouldn't it be easier to do `...swcCliOptions` instead?

Reply:
Indeed destructuring is easier to maintain defaults, changed and committed